platform/security/platform-security-session/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -264,4 +270,16 @@ public void renewSessionId(String oldClusterId, String oldNodeId, HttpServletReq
             }
         }
     }
+
+    public static byte[] getBcRbgSeed() {
+        EntropySourceProvider esp = new BasicEntropySourceProvider(new SecureRandom(), true);
+        DualECSP800DRBG bcRbg = new DualECSP800DRBG(new SHA256Digest(),
+                256,
+                esp.get(256),
+                null,
+                null);
```

Comment:
It might be worth it to use a nonce instead of null. You can just call the generate seed method of SecureRandom to produce you some random bytes for the seed. That should be sufficient.